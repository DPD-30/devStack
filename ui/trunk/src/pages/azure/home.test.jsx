import React from 'react'
import { renderToString } from 'react-dom/server'

import PagesAzureHome from 'pages/azure/home.jsx'

jest.unmock('pages/azure/home.jsx')


describe('', () => {
    it('PagesAzureHome should match snapshot', () => {
        const comp = renderToString(<PagesAzureHome />)

        expect(comp).toMatchSnapshot()
    })
})
