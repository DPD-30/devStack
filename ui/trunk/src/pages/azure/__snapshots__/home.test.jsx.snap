// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PagesAzureHome should match snapshot 1`] = `
<div data-reactroot>
  Azure user home
</div>
`;
